resources/js/src/app/components/item/singleitem/FeedbackListEntry.vue

```diff
@@ -80,41 +80,54 @@
 
     <div v-if="!!editableFeedback && !isReply">
       <div class="stars">
-        <template v-for="i in [5,4,3,2,1]">
-          <input
-            :id="'star-' + _uid + '-' + i"
-            :key="'stars_input_' + i"
-            v-model="editableFeedback.ratingValue"
-            :class="'star star-' + i"
-            type="radio"
-            :value="i"
-            name="ratingValue"
-          >
-          <label
-            :key="'stars_label_' + i"
-            :class="'star star-' + i"
-            :for="'star-' + _uid + '-' + i"
-          />
-        </template>
+        <fieldset>
+          <legend><span class="d-none">{{ $translate("Feedback::Feedback.feedbackTextLegend") }}</span></legend>
+          <template v-for="i in [5,4,3,2,1]">
+            <input
+              :id="'star-' + _uid + '-' + i"
+              :key="'stars_input_' + i"
+              v-model="editableFeedback.ratingValue"
+              :class="'star star-' + i"
+              type="radio"
+              :value="i"
+              name="ratingValue"
+            >
+            <label
+              :key="'stars_label_' + i"
+              :class="'star star-' + i"
+              :for="'star-' + _uid + '-' + i"
+            ><span class="d-none">{{ $translate("Feedback::Feedback.feedbackAverageLabel") }}</span></label>
```

Comment:
1. it's literally hidden from screenreaders as well
Makes more sense to use visually-hidden again?

2. It would be the same label for every star
maybe something like "ratingStarLabel": "{stars} of 5 Stars" would make more sense